auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailActivity.java

```diff
@@ -20,7 +20,6 @@
 import android.support.annotation.RestrictTo;
 import android.support.design.widget.TextInputLayout;
 import android.support.v4.app.FragmentTransaction;
-import android.view.View;
```

Comment:
For the 6th time: We need minimal changes, no refactoring. Reformatting, changing imports etc. IS refactoring.